File: 10.prime_and_composite_numbers/lesson_10_min_perimeter_rectangle.py
def solution(N):
    sqrt = int(N**(1/2))
    mini = 2 * (1 + N)
    for i in range(sqrt, 1, -1):
        if(N % i == 0):
            mini = min(mini, 2 * (i + int(N / i)))
    return mini

File: 10.prime_and_composite_numbers/lesson_10_min_perimeter_rectangle.js
function solution(N) {
    let perimeter = 2 * (1 + N)
    for(let i = 1; i <= Math.sqrt(N); i++) {
        if(N % i == 0) {
            perimeter = Math.min(perimeter, 2 * (i + (N / i)))
        }
    }
    return perimeter
}